predicthq/endpoints/v1/broadcasts/schemas.py

```diff
@@ -0,0 +1,159 @@
+from predicthq.endpoints.schemas import (
+    PaginatedMixin, SortableMixin,
+    BooleanType, DateTimeRange, DateTimeType, FloatType, IntRange, IntType,
+    ListType, Model, ModelType, ResultSet, ResultType, StringType
+)
+
+
+class BroadcastEventParams(Model):
+
+    class Options:
+        serialize_when_none = False
+
+    event_id = ListType(StringType)
+    category = ListType(StringType)
+    label = ListType(StringType)
+
+
+class BroadcastLocationParams(Model):
+
+    class Options:
+        serialize_when_none = False
+
+    place_id = ListType(StringType)
+
+
+class SearchParams(PaginatedMixin, SortableMixin, Model):
+
+    class Options:
+        serialize_when_none = False
+
+    broadcast_id = ListType(StringType)
+    location = ModelType(BroadcastLocationParams)
+    phq_viewership = ModelType(IntRange)
+    start = ModelType(DateTimeRange)
```

Comment:
Using DateTimeRange as-is for start and updated filters is problematic because the API only accepts `YYYY-MM-dd` or `YYYY-MM-ddTHH:mm:ss` formats for times, but DateTimeRange seems to (by default) format to include fractions of a second too, e.g. '2020-01-31T23:59:59.999999'

Reply:
Updated the API and tested the sdk, works ok now.